odl/solvers/util/callback.py

```diff
@@ -25,6 +27,62 @@
            'CallbackProgressBar')
 
 
+def call_callback(callback, x, **kwargs):
```

Comment:
Why do we need this function? Why not just `callback(x, **kwargs)`? I.e. why do we need to filter the `kwargs`?